microscopy_png/participants.tsv

```diff
@@ -0,0 +1,3 @@
+participant_id	testParticipant
+sub-rat2	test-rat-2
```

Comment:
Will the splits be done on participants or samples? If the split is done on participants, having 3 subjects can be convenient (1 for training, 1 for validation, and 1 for testing). 2 participants will work, it will only be necessary to change the parameter "test_fraction": 0.0, when/if testing training with microscopy data.

Reply:
Split could be done on either subject or sample. I tried to keep the dataset as light as possible. I think we should duplicate files in the tests if we need more subjects (as it is done with MRI).

Reply:
In this line of thought, why not create the dataset with a single patient as it is done with MRI?

Reply:
I wanted to add files for the "sessions" and "scans" architectures. It seemed clearer to me to do it directly on the dataset than "buried" in a test somewhere.  I made sure than the dataset is under 3Mb though.